src/pmlogger/pmlogger_check.sh

```
@@ -534,7 +540,14 @@ _check_archive()
 	    :
 	else
 	    logdir=`dirname "$logfile"`
-	    echo "Directory (`cd "$logdir"; $PWDCMND`) contents:"
+	    __pwddir=`cd "$logdir"; $PWDCMND`
```

Shellcheck (0.11.0)

[style] Use $(...) notation instead of legacy backticks `...`.

(SC2006)